src/components/expanded-state/SupportedCountriesExpandedState.js

```diff
@@ -1,39 +1,111 @@
-import React from 'react';
 import { values } from 'lodash';
-import { StyleSheet, Text } from 'react-native';
-import FloatingPanels from './FloatingPanels';
-import { AssetPanel } from './asset-panel';
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { useDimensions } from '../../hooks';
+import isNativeStackAvailable from '../../helpers/isNativeStackAvailable';
 import { supportedCountries } from '../../references/wyre/supportedCountries';
-import { Centered } from '../layout';
-import { padding } from '../../styles';
+import { colors, fonts, padding } from '../../styles';
 import { deviceUtils } from '../../utils';
+import { ButtonPressAnimation } from '../animations';
+import { FloatingEmojis, FloatingEmojisTapHandler } from '../floating-emojis';
+import { Centered } from '../layout';
+import { Text } from '../text';
+import { AssetPanel } from './asset-panel';
+import FloatingPanels from './FloatingPanels';
 
 const sx = StyleSheet.create({
-  text: {
-    fontSize: 13,
-    fontWeight: 'normal',
-    lineHeight: 20,
+  body: {
+    color: colors.alpha(colors.blueGreyDark, 0.6),
+    fontSize: parseFloat(fonts.size.smedium),
+    lineHeight: 22,
+    textAlign: 'center',
+  },
+
+  footer: {
+    fontSize: parseFloat(fonts.size.smedium),
+    marginTop: 12,
     textAlign: 'center',
   },
 
   title: {
+    fontSize: parseFloat(fonts.size.large),
     fontWeight: 'bold',
-    marginBottom: 20,
+    marginBottom: 10,
+    textAlign: 'center',
   },
 });
 
-const countries = values(supportedCountries).map(c => c.name);
-const countriesList = `${countries.join(', ')}.`;
+const countries = values(supportedCountries).map(c =>
+  c.name == 'United States'
+    ? 'United\xa0States (except CT, HI, NC, NH, NY, VA, VT)'
+    : c.name.replace(/ /g, '\xa0')
+);
+const countriesList = `${countries.join(', ')}`;
+
+const emojiArray = [
+  'flag-ch',
+  'flag-at',
+  'flag-be',
+  'flag-ca',
+  'flag-cz',
+  'flag-dk',
+  'flag-ee',
+  'flag-fi',
+  'fr',
+  'de',
+  'flag-gr',
+  'flag-hk',
+  'flag-ie',
+  'it',
+  'flag-lv',
+  'flag-lt',
+  'flag-lu',
+  'flag-mx',
+  'flag-nl',
+  'flag-nz',
+  'flag-no',
+  'flag-pl',
+  'flag-pt',
+  'flag-sk',
+  'flag-si',
+  'es',
+  'flag-au',
+  'us',
+];
 
 const SupportCountriesExpandedState = () => {
+  const { isTallPhone } = useDimensions();
+  const modalMargin =
+    (isTallPhone ? 90 : 45) + (isNativeStackAvailable ? 10 : 0);
+
   return (
-    <FloatingPanels maxWidth={deviceUtils.dimensions.width - 110}>
-      <AssetPanel>
-        <Centered css={padding(24, 25)} direction="column">
-          <Text style={sx.title}>List of supported countries</Text>
-          <Text style={sx.text}>{countriesList}</Text>
-        </Centered>
-      </AssetPanel>
+    <FloatingPanels
+      marginBottom={modalMargin}
+      maxWidth={Math.min(270, deviceUtils.dimensions.width - 100)}
```

Comment:
I'd move this into a const outside of the function so it doesn't recalculate on every render